.github/workflows/CD.yaml

```diff
@@ -104,10 +104,15 @@ jobs:
           KEYCLOAK_UPLOADER_PASSWORD: ${{ secrets.KEYCLOAK_UPLOADER_PASSWORD }}
           KEYCLOAK_GOOGLE_SECRET: ${{ secrets.KEYCLOAK_GOOGLE_SECRET }}
           KEYCLOAK_GOOGLE_ID: ${{ secrets.KEYCLOAK_GOOGLE_ID }}
+          PROXY_LETSENCRYPT_PATH: ${{ secrets.PROXY_LETSENCRYPT_PATH }}
+          PROXY_NGINX_URLS: ${{ secrets.PROXY_NGINX_URLS }}
+          PROXY_LETSENCRYPT_ARGS: ${{ secrets.PROXY_LETSENCRYPT_ARGS }}
+      - name: Prepare for check deployment
+        run: sudo apt update && sudo apt install -y upower
```

Comment:
wofür ist upower? wofür sind PROXY_NGINX_URLS?

Reply:
UPower ist eine library, die Fehlermeldungen in Cypress erzeugt, wenn diese nicht da ist. 
![image](https://user-images.githubusercontent.com/108792843/178944224-d402d6f8-2c0a-429f-aa13-6d95db641383.png)
Weiß aber nicht, ob die Fehlermeldungen fatal sind. Kann auch gerne nochmal versuchen, ob die CD auch ohne den install funktioniert.